src/sentry/snuba/outcomes.py

```diff
@@ -263,40 +271,52 @@ def __init__(
             query_groupby.update(groupby.get_snuba_groupby())
         self.query_groupby = list(query_groupby)
 
-        self.conditions, self.filter_keys = get_filter(query, params)
+        self.group_by = []
+        for key in self.query_groupby:
+            self.group_by.append(Column(key))
+
+        conditions, filter_keys = get_filter(query, params)
```

Comment:
want to modify the `get_filter` method to be called `get_conditions`, and do all this business below with conditions in there?